loops.js

```diff
@@ -18,8 +21,12 @@
 // klaar!
  // ==========================================
 
-
-
+for (j = 0; j < 4; j++) {
```

Comment:
De variable wordt niet gedefinieed met ' let '
Zou de variable van een for loop bij i laten beginnen dan j en dan k